### DISCUSSION TITLE
번아웃을 겪고 남겨 놓는 기록 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#47] [Announcements]
# 번아웃을 겪고 남겨 놓는 기록 - 정현수 기술 블로그

.

https://junghyeonsu.com/posts/2023-burn-out-retrospect/

**Comment by dk-kling** [comment]
항상 응원하고 있습니다 :)

**Comment by JoonheeJeong** [comment]
보고 싶은 현수야~ 열심히 달려온 만큼 중간에 마음고생도 겪었구나 ㅜㅜ 아픈 만큼 더 성장할 거야! 화이팅🤗🙏🏼

**Reply by junghyeonsu** [reply]
준희 잘 지내지? 고마워 형ㅠ 잘 지내고 형도! 백엔드 개발한다고 들었는데 형도 화이팅!

**Comment by alalshow** [comment]
응원합니다!!
